package.json

```diff
@@ -23,14 +23,14 @@
     "test-local": "npm run test-browser && npm run test-android && npm run test-ios",
     "test-local-on-windows": "npm run test-browser && npm run test-android",
     "test-android": "node main.js --platform android --plugin ./spec/testable-plugin/",
-    "test-ios": "node main.js --platform ios --plugin ./spec/testable-plugin/ --args=--buildFlag='-UseModernBuildSystem=0' --verbose",
+    "test-ios": "node main.js --platform ios --plugin ./spec/testable-plugin/ --target \"iPhone-X, 11.1\" --verbose",
```

Comment:
Couuld we get rid of that if we only test with Xcode 10.2?

Reply:
From previous tests, this is what I saw.
- PASS `Xcode 10.0` discovered, built, and launched simulator for `iPhone-XS-Max, 12.0`.
- FAIL `Xcode 10.1` discovered `iPhone-XR, 12.0` but could not find the simulator so it built of `iPhone X`. Then it tried to launch the XR simulator `--devicetypeid com.apple.CoreSimulator.SimDeviceType.iPhone-XR` but failed again to find.
- PASS `Xcode 10.2` discovered, built, and launched simulator for `iPhone-XR, 12.0`.

So if we dropped then yes it would pass. BUT IMO I think we should make `target` a required property. At least for iOS. I believe the fuzzy logic to automatically pick some device for a user is not always stable. The list that selects the device from has been ordered wrong once before and picked an unsupported device and OS version to our platform.

Reply:
See https://github.com/apache/cordova-ios/issues/602 for the Xcode 10.1 problem
See https://github.com/apache/cordova-ios/issues/601 for the Xcode 9.x problem

Reply:
As this CI configuration is the template for all plugins, I think we should be fine with going with just Xcode 10.2 - otherwise we just add a lot of runtime to all our tests that is not really necessary.